packages/bpk-mixins/src/mixins/_buttons.scss

```diff
@@ -41,17 +41,20 @@ $bpk-spacing-v2: true;
   border-radius: $bpk-button-border-radius;
   background-image: linear-gradient(
     -180deg,
-    $bpk-button-gradient-start-color 0%,
-    $bpk-button-gradient-end-color 100%
+    $bpk-private-button-primary-normal-background-day 0%,
+    $bpk-private-button-primary-normal-background-day 100%
   );
   background-image: linear-gradient(
     -180deg,
     var(
         --bpk-button-primary-gradient-start-color,
-        $bpk-button-gradient-start-color
+        $bpk-private-button-primary-normal-background-day
       )
       0%,
-    var(--bpk-button-primary-gradient-end-color, $bpk-button-gradient-end-color)
+    var(
+        --bpk-button-primary-gradient-end-color,
+        $bpk-private-button-primary-normal-background-day
+      )
       100%
   );
```

Comment:
From memory we can most likely remove setting the background-image as a linear-gradient because we only use one solid colour for the button - Blue.

This is legacy from our old buttons where the colour was made of two different shades of green!